go/protocol/chat1/extras.go

```diff
@@ -420,6 +420,7 @@ func (m MessageUnboxedValid) Etime() gregor1.Time {
 }
 
 func (m MessageUnboxedValid) RemainingEphemeralLifetime(now time.Time) time.Duration {
+	now = keybase1.ForceWallClock(now)
 	remainingLifetime := m.Etime().Time().Sub(now).Round(time.Second)
```

Comment:
I think the original bug happened here, if the `now` that was passed in was well in the past compared to the wall clock time.

Reply:
:+1: